anta/reporter/__init__.py

```diff
@@ -27,6 +28,18 @@
 class ReportTable:
     """TableReport Generate a Table based on TestResult."""
 
+    class Headers(BaseModel):
```

Comment:
Should we use a `dataclass` here to be able to use like `headers.number_of_success`? I think it would be cleaner.

Reply:
Convert all `class Headers` to `dataclass`. But still per report type since there is no common headers across all reports